packages/core/src/extension/extension-selector.ts

```diff
@@ -19,6 +20,11 @@ export function createExtensionSelectorFactory(): ExtensionSelectorFactory {
         (data) => () => data,
     );
 
+    const getExtensionByRegion = createSelector(
+        (state: ExtensionState) => state.data,
+        (data) => (region: ExtensionRegion) => data?.filter((e) => e.region === region)[0],
```

Comment:
Can we use `find` here, it will return the first element that is truthy.

Reply:
Good point. Code updated. Thanks.